src/main/java/pl/bartek/parking/service/ParkingStayStartRequest.java

```diff
@@ -0,0 +1,20 @@
+package pl.bartek.parking.service;
+
+public class ParkingStayStartRequest {
+
+    private String vehicleRegistrationNumber;
+    private String customerType;
+
+    public ParkingStayStartRequest(String vehicleRegistrationNumber, String customerType) {
```

Comment:
Z jednej strony tak, ale nie lubię wrzucać za dużo logiki do kontrolera, więc wolałem przyjąć żądanie "as is" i przekazać dalej do serwisu - niech on się martwi co z nim zrobić. Później wystarczy tylko przetestować serwis, bez kontrolera bo tam logiki nie ma :)